shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -1,10 +1,23 @@
-use cassandra_cpp::{stmt, Error, Session, Value, ValueType};
+use cassandra_cpp::{stmt, Cluster, Error, Session, Value, ValueType};
 use ordered_float::OrderedFloat;
 
 mod basic_driver_tests;
 
+#[allow(unused)]
+pub fn cassandra_connection(contact_points: &str, port: u16) -> Session {
```

Comment:
Isnt this now completely duplicated with the existing method?

Reply:
Yes

Reply:
yeah understandable to not blow up this PR, but at least make the other cassandra_connection call into this one?

Reply:
Ended up moving stuff around a bit to make this change, so I opened another PR to make the change a bit easier: #584 584

Reply:
perfect